pages/qdm-to-qicore.md

```diff
@@ -1612,7 +1552,7 @@ indicating information about medications that have been dispensed.
 | setting                       | [MedicationDispense.category](StructureDefinition-qicore-medicationdispense-definitions.html#MedicationDispense.category)                                                          | Inpatient, Outpatient, Community, Discharge                                                                                      |
 | reason                        | [MedicationDispense.statusReason\[x\]](StructureDefinition-qicore-medicationdispense-definitions.html#MedicationDispense.statusReason[x])                                          | The reason for ordering or not ordering the medication                                                                           |
 | relevant dateTime             | [MedicationDispense.whenHandedOver](StructureDefinition-qicore-medicationdispense-definitions.html#MedicationDispense.whenHandedOver)                                              | When provided to patient or representative                                                                                       |
-| relevant Period               | [MedicationDispense.extension:validityPeriod](StructureDefinition-qicore-medicationdispense-definitions.html#MedicationDispense.extension:validityPeriod)                          |                                                                                                                                  |
+| relevant Period               | [MedicationDispense.daysSupply](StructureDefinition-qicore-medicationdispense-definitions.html#MedicationDispense.daysSupply)                          |                                                                                                                                  |
```

Comment:
How does this work? daysSupply is a quantity, not a period.

Reply:
I've relayed the question to Floyd on this. This is actually part of a bigger conversation I'm having with Floyd. We may end up removing timing information from Medications, or fixing timing in another way.

Reply:
ok, actually this will be changed to "N/A" in the next commit